src/integrationTest/resources/log4j2-test.properties

```diff
@@ -28,17 +28,20 @@ logger.auditlogs.level = info
 # Logger required by test org.opensearch.security.http.JwtAuthenticationTests
 logger.httpjwtauthenticator.name = com.amazon.dlic.auth.http.jwt.HTTPJwtAuthenticator
 logger.httpjwtauthenticator.level = debug
+logger.backendreg.additivity = false
```

Comment:
What does this log4j setting do?

Reply:
From what I understand, additivity false means that logs will not be appended to its ancestors. It will be for "my" module only
https://stackoverflow.com/questions/22692097/what-is-the-result-of-making-log4j-additivity-equals-to-true-or-false

Reply:
My question, is why do we need this setting for these 3 components?

Reply:
This prevents these log statements from being send to the console logger which is the default logger, but still captures them in a way that can be used by the log capturing listeners that are used in some tests cases.

Reply:
Those debug level statements aren't that useful for analyzing test failure/success, there is a decently reduction in how much logout needs to be scanned through:

### integration-tests gradle output
- Before 21,400 lines
- After 13,783 lines

### resource-tests gradle output
- Before 10,366 lines
- After 640 lines

Reply:
> My question, is why do we need this setting for these 3 components?

@DarshitChanpura See the commend on line 28

Reply:
thank you for your explanation @peternied !